### PR TITLE
Update version to 0.3.15

### PR DESCRIPTION
**Pull request by ericliuche**

### CODE REVIEW

File: airplane/display.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def text(content: str, dedent: bool = True) -> None:
 
 
 @deprecation.deprecated(
-    deprecated_in="0.3.14",
+    deprecated_in="0.3.15",
```
Comment (ameyakhare) [inline airplane/display.py:31]: why is this changed?

Comment (ericliuche) [reply airplane/display.py:31]: oops my b, bad find and replace
